tslint.json

```diff
@@ -16,7 +16,8 @@
     "label-position": true,
     "new-parens": true,
     "no-angle-bracket-type-assertion": true,
-    "no-any": true,
+    // no-any is a pain when working with untyped dependencies in .js, such as our ui components.
+    // "no-any": true,
```

Comment:
I'm not sure we want to remove this type check. I realize it's not convenient for JS, but we can use directives to ignore safe use of any.

Reply:
Reverted. Previously I had any in the middle of HTML and the directive didn't work. But the new structure works fine now.